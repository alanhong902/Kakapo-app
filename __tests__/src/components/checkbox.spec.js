import React from 'react';
import renderer from 'react-test-renderer';
import Checkbox from 'components/ui/checkbox/checkbox';

test('<Checkbox/>', () => {
  const props = {
    checked: false,
    dispatch: e => e,
    handleChange: () => 'checkbox changed!',
    label: 'testLabel',
    name: 'testName',
  };
  const tree = renderer.create(<Checkbox {...props} />).toJSON();
  expect(tree).toMatchSnapshot();
});
