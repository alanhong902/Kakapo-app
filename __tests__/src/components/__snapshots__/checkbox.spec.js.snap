// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Checkbox/> 1`] = `
<label
  className="switch"
  htmlFor="testName"
>
  <span
    className="switch-label"
  >
    testLabel
  </span>
  <input
    checked={false}
    id="testName"
    name="testName"
    onChange={[Function]}
    type="checkbox"
    value={false}
  />
</label>
`;
